packages/block-library/src/post-comment-avatar/edit.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	InspectorControls,
+	__experimentalUseBorderProps as useBorderProps,
+	__experimentalImageSizeControl as ImageSizeControl,
+	useBlockProps,
+} from '@wordpress/block-editor';
+import { PanelBody, ResizableBox } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __, isRTL } from '@wordpress/i18n';
+
+export default ( { attributes, context, setAttributes } ) => {
+	const { className, style, height, width } = attributes;
+	const { commentId } = context;
+
+	// Maximum provided by the WordPress site.
+	const maxImageWidth = 96;
```

Comment:
I haven't checked this, by you might want to make sure there is no php filter for this value. WP has tons of filters 😄

Reply:
What is the API to run PHP filters from JS?

Reply:
You have a get_avatar hook that sets the size: https://developer.wordpress.org/reference/hooks/get_avatar/

Is this affecting the `edit.js` code, right?

Reply:
I removed this code, instead, now we use the default values of `block.json` file. I hope that will be a better approach!